src/flows/pipes/RawFluxEditor/view.tsx

```diff
@@ -147,27 +119,33 @@ const Query: FC<PipeProp> = ({Context}) => {
     />
   )
 
-  return (
-    <Context controls={controls}>
-      <Suspense
-        fallback={
-          <SpinnerContainer
-            loading={RemoteDataState.Loading}
-            spinnerComponent={<TechnoSpinner />}
+  return useMemo(
```

Comment:
Twas failing tests due to timeout. Was rendering 3 times on load.
Got it down to 2 loads (the minimum possible with the editor context callback) --> tests are now passing.